### PR TITLE
Update REQUIREMENTS.md

### PR DESCRIPTION
**Pull request by imjasonh**
Numbering requirements makes them easier to reference (lol) by shorthand, e.g., **Filtering-3**, **Backcompat-6**, etc.

### CODE REVIEW

File: docs/REQUIREMENTS.md [hunk 1]
```diff
@@ -16,35 +16,35 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
...
-- As a user, I want to query a registry for all stored artifacts of a particular type that reference another artifact by its tag and digest.
-- As a user, I want to query a registry for all stored artifacts based on annotations that reference another artifact by its tag and digest.
-- As a user, I want to fetch the most up-to-date artifact, collection of artifacts, or application.
-- As an artifact producer, I would like to reduce the number of tags that reference artifacts.
+1. As a user, I want to query a registry for stored artifacts by digest and tag.
+1. As a user, I want to query the registry for all stored artifacts that reference another artifact by its tag and digest.
+1. As a user, I want to query a registry for all stored artifacts of a particular type that reference another artifact by its tag and digest.
+1. As a user, I want to query a registry for all stored artifacts based on annotations that reference another artifact by its tag and digest.
+1. As a user, I want to fetch the most up-to-date artifact, collection of artifacts, or application.
+1. As an artifact producer, I would like to reduce the number of tags that reference artifacts.
```
Comment (jdolitsky) [inline docs/REQUIREMENTS.md:24]: "I'm number 1!"

**Review by jdolitsky — CHANGES_REQUESTED** [review]

**Review by SteveLasker — APPROVED** [review]
LGTM markdown formatting.

File: docs/REQUIREMENTS.md (hunk 1 above)
Comment (imjasonh) [reply docs/REQUIREMENTS.md:24]: Would you prefer that the Markdown numbers them explicitly? It ends up getting rendered as incrementing numbers either way.

Comment (jdolitsky) [reply docs/REQUIREMENTS.md:24]: oh neat! didnt know that

**Review by jdolitsky — APPROVED** [review]

**Pull request merged**